src/main/kotlin/no/nav/tiltaksarrangor/service/TilgangskontrollService.kt

```diff
@@ -0,0 +1,47 @@
+package no.nav.tiltaksarrangor.service
+
+import no.nav.tiltaksarrangor.model.exceptions.SkjultDeltakerException
+import no.nav.tiltaksarrangor.model.exceptions.UnauthorizedException
+import no.nav.tiltaksarrangor.repositories.model.AnsattDbo
+import no.nav.tiltaksarrangor.repositories.model.DeltakerDbo
+import no.nav.tiltaksarrangor.repositories.model.DeltakerMedDeltakerlisteDbo
+import org.springframework.stereotype.Service
+
+@Service
+class TilgangskontrollService(
```

Comment:
Lurt å lage en egen service for dette! Skjønner godt hvis du ikke vil det nå, men det ligger en del tilgangsfunksjonalitet i ansattservicen, f.eks. harTilgangTilDeltaker som kanskje burde ha væt flyttet hit, men vi kan gjøre det senere også.

Reply:
Tror jeg tar det litt etter litt, det er nok en del mer som kan refaktoreres.